src/Umbraco.Core/Services/IMemberService.cs

```diff
@@ -199,6 +200,14 @@ public interface IMemberService : IMembershipMemberService
         /// <param name="memberTypeId">Id of the MemberType</param>
         void DeleteMembersOfType(int memberTypeId);
 
+        /// <summary>
+        /// Exports member data based on their unique Id
+        /// </summary>
+        /// <param name="key">The unique <see cref="Guid">member identifier</see></param>
+        /// <param name="currentUser">The <see cref="IUser">user</see> requesting the export</param>
+        /// <returns><see cref="HttpResponseMessage"/></returns>
+        HttpResponseMessage ExportMemberData(Guid key, IUser currentUser);

```

Comment:
Ack! why is this on the IMemberService? The `IMemberService` is responsible for speaking to the DB, it should not be aware of anything to do with `HttpResponseMessage`